Sources/MapboxNavigation/NavigationView.swift

```diff
@@ -147,6 +147,12 @@ open class NavigationView: UIView {
     
     lazy var resumeButton: ResumeButton = .forAutoLayout()
     
+    var wayNameViewLayoutGuide: UILayoutGuide? {
```

Comment:
This property provides the ability to change default `WayNameView` position. Intention is to make it public after more testing.